### PR TITLE
SIG-Windows 2021 Annual Report

### PR DESCRIPTION
**Pull request by jayunit100**
annual report for sig-windows.

### CODE REVIEW

**Comment by marosset** [comment]
Thanks @jayunit100 
 Related to #5518

File: sig-windows/annual-report-2020 [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+Alpha / going alpha
+Privileged containers
+Node log viewer
+Beta / going beta
+DSR support
+Stable / going stable
+CSI proxy 
+
+What areas and/or subprojects does the group need the most help with?
+
```
Comment (aravindhp) [inline sig-windows/annual-report-2020:78]: I suggest adding API reviews and `sig-auth`.

**Review by aravindhp — COMMENTED** [review]
Thanks for doing this, @jayunit100

**Comment by liggitt** [comment]
cc @kubernetes/steering-committee

File: sig-windows/annual-report-2020 [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+[ ] titled: annual-report-YEAR.md
+[ ] are there any responses that you’d like to share privately first? steering-private@kubernetes.io or tag your liaison in for discussion. 
+
+# Special Interest Groups:
+(do not copy if you are responding for a working group)
+
+
+# Instructions for this section:
+[ ] each response should have supporting documentation, linked KEPs, graphs, data
+[ ] err on the side of being descriptive with your responses. End users and others that read this might not know our upstream acronyms, shorthand, or where to find a reference.
```
Comment (parispittman) [inline sig-windows/annual-report-2020:20]: you can remove everything from this line up - row 1:20

File: sig-windows/annual-report-2020 [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+How are you doing with operational tasks in sig-governance.md?
+Is your README accurate?have a CONTRIBUTING.md file?
+Yes / Yes - README.md is up to date, CONTRIBUTING.md is still accurate but could use some updates.  
+
+All subprojects correctly mapped and listed in sigs.yaml?
+Yes
+
+What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+Meetings have decent attendance but few active participants.
```
Comment (parispittman) [inline sig-windows/annual-report-2020:31]: what's an active participant in this case? someone who contributes code/docs to GitHub or active attendees of the meetings?

File: sig-windows/annual-report-2020 [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+
+What’s your meeting culture? Large/small, active/quiet, learnings? Meeting notes up to date? Are you keeping recordings up to date/trends in community members watching recordings?
+Meetings have decent attendance but few active participants. 
+Meeting notes are used during the meetings are up to date. 
+Recordings are up to date but few views 
+
+How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+Owners files are up to date. Existing subprojects are mostly in maintenance mode. We would like to try starting one or two subprojects to gather more interest / new participation in SIG-Windows.
+Same question as above but for working groups.
+N/A
```
Comment (parispittman) [inline sig-windows/annual-report-2020:38]: How does the group get updates from the subproject owners? are there any new subprojects in the last year? and then the owners file question, too.

File: sig-windows/annual-report-2020 [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+Membership
+
+Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+Yes
+
+How do you measure membership? By mailing list members, OWNERs, or something else?
+We don’t measure membership.
+
+How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+We are blocked on reviewer/approver bandwidth from other SIGs much more often than reviewer/approver bandwidth in our SIG.
```
Comment (parispittman) [inline sig-windows/annual-report-2020:51]: ty for calling this out

File: sig-windows/annual-report-2020 [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+Is there a healthy onboarding and growth path for contributors in your SIG? What are some activities that the group does to encourage this? What programs are you participating in to grow contributors throughout the contributor ladder?
+We want to focus on this in the next few months. Some ideas we have are pairing with folks and establishing one or two new focused subprojects that will encourage new participants.
+
+What programs do you participate in for new contributors? We’re wanting to groom long-term/dedicated contributors. Are there programs to help with this? Does the group have contributors from multiple companies/affiliations? Can end users/companies contribute in some way that they currently are not?
```
Comment (parispittman) [inline sig-windows/annual-report-2020:55]: need a line break after the contributors question

Comment (parispittman) [reply sig-windows/annual-report-2020:55]: and then question for long-term/dedicated contributors - to a specific subproject? what kind of contributors? windows experts? can you elaborate here?

File: sig-windows/annual-report-2020 [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+Yes, we have contributors from many companies. Current initiatives and project health
+
+[ ] Please include links to KEPs and other supporting information that will be beneficial to multiple types of community members. 
+
+What are initiatives that should be highlighted, lauded, shout out, that your group is proud of? Currently underway? What are some of the longer tail projects that your group is working on?
+
+Containerd support
```
Comment (parispittman) [inline sig-windows/annual-report-2020:62]: what section do these belong in? proud of, shout out? long tail? or currently underway?

File: sig-windows/annual-report-2020 [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+Beta / going beta
+DSR support
+Stable / going stable
+CSI proxy 
+
+What areas and/or subprojects does the group need the most help with?
+
+E2e tests / coverage / keeping things green
+Testgrid
+Cross-SIG collaboration
```
Comment (parispittman) [inline sig-windows/annual-report-2020:81]: on what? what sigs specifically?

**Review by parispittman — CHANGES_REQUESTED** [review]

**Comment by liggitt** [comment]
@parispittman's comments covered the questions I had. Looks like there were a couple other suggestions to pull in, and preview/tweak formatting where needed.

File: sig-windows/annual-report-2020 (hunk 1 above)
Comment (jayunit100) [reply sig-windows/annual-report-2020:60]: done thanks

**Comment by jayunit100** [comment]
finished updating ! thanks

File: sig-windows/annual-report-2020 [hunk 9]
```diff
@@ -0,0 +1,69 @@
+# Operational
```
Comment (liggitt) [inline sig-windows/annual-report-2020:1]: please rename to a .md suffix so this renders as markdown

Comment (liggitt) [reply sig-windows/annual-report-2020:1]: once you do that, view in the markdown rendering and add newlines (and maybe bold the questions) to make the answers legible and separate from the questions.

File: sig-windows/annual-report-2020 [hunk 10]
```diff
@@ -0,0 +1,69 @@
...
+Meeting notes are used during the meetings are up to date. 
+Recordings are up to date but few views 
+
+How does the group get updates, reports, or feedback from subprojects? Are there any springing up or being retired? Are OWNERS files up to date in these areas?
+Owners files are up to date. Existing subprojects are mostly in maintenance mode. We would like to try starting one or two subprojects to gather more interest / new participation in SIG-Windows.
+Same question as above but for working groups.
+We don't get updates from subprojects and are in the project of refactoring subprojects to match where active development is happening.
+
+When was your last monthly community-wide update? (provide link to deck and/or recording)
+Sept 17 Slides recording
```
Comment (liggitt) [inline sig-windows/annual-report-2020:21]: link to slides and recording?

File: sig-windows/annual-report-2020.md [hunk 11]
```diff
@@ -18,7 +18,7 @@ Same question as above but for working groups.
 We don't get updates from subprojects and are in the project of refactoring subprojects to match where active development is happening.
 
 When was your last monthly community-wide update? (provide link to deck and/or recording)
-Sept 17 Slides recording
+No links available, we may modify this later.
```
Comment (marosset) [inline sig-windows/annual-report-2020.md]: Links 
- Recording: https://www.youtube.com/watch?v=6Wn_dIEg0E8
- Slides: https://docs.google.com/presentation/d/1FkS6FIes9opMR1B4qXyVzwGfUBnX0A-sh27Kcnyi-e4/edit#slide=id.g97ab5232db_0_0

**Comment by jayunit100** [comment]
all comms addresssed

**Pull request merged**